library/backend.cpp

```diff
@@ -436,7 +436,7 @@ static int dnet_cmd_backend_control_dangerous(struct dnet_net_state *st, struct
 		return -ENOENT;
 	}
 
-	if (cmd->size != sizeof(dnet_backend_control) + control->ids_count * sizeof(dnet_raw_id)) {
+	if (cmd->size < sizeof(dnet_backend_control) + control->ids_count * sizeof(dnet_raw_id)) {
```

Comment:
Now the check isn't quite troubleproof, because each control request can have tail of any length. JFMI: what is planned to add to request?

Reply:
> what is planned to add to request?

`chunks_dir` in https://github.com/interiorem/elliptics/pull/264